parser-typechecker/src/Unison/Codebase/Editor/HandleInput.hs

```diff
@@ -636,9 +636,7 @@ loop = do
           case uf of
             Nothing -> do
               let parseNames0 = (`Names3.Names` mempty) basicPrettyPrintNames0
-                  -- use suffixed names for resolving the argument to display
-                  parseNames = Names3.suffixify parseNames0
-                  results = Names3.lookupHQTerm hq parseNames
+                  results = Names3.lookupHQTerm hq parseNames0
```

Comment:
`lookupHQTerm` does suffix-based lookup now, so no need to suffixify the `Names` passed in.